fetch.bs

```diff
@@ -1825,14 +1825,14 @@ run these steps:
 </ol>
 
 
-<h3 id=client-hints-list>Client hints list</h3>
+<h3 id=client-hints-list>Client-hints set</h3>
 
-<p class=note>This section will be integrated into HTTP Client Hints.
+<p class=XXX>This section will be integrated into HTTP Client-Hints.
```

Comment:
Sorry for being unclear, this issue marker should indicate that Client Hints integration with Fetch is far from finished, similar to the issue marker you added below. I was thinking it would be good to stress that in multiple places.

Reply:
Changed to point at https://github.com/whatwg/fetch/issues/726
Let me know if that works